### PR TITLE
Handle case when version and latest_version are not present in the data

### PR DESCRIPTION
**Pull request by tuxdna**
Fixes: https://github.com/fabric8-analytics/fabric8-analytics-data-model/issues/103

### CODE REVIEW

File: src/graph_populator.py [hunk 1]
```diff
@@ -37,7 +40,7 @@ def construct_version_query(cls, input_json):
         """Construct the query to retrieve detailed information of given version of a package."""
         pkg_name = input_json.get('package')
         ecosystem = input_json.get('ecosystem')
-        version = cls._sanitize_text_for_query(input_json.get('version'))
+        version = cls._sanitize_text_for_query(input_json.get('version') or '')
```
Comment (jpopelka) [inline src/graph_populator.py]: Why do we need the `or ''` here ?

With the newly added
```python
if text is None:
    return ''
```
both the `_sanitize_text_for_query(None)` and `_sanitize_text_for_query('')` return `''`, don't they ?

File: src/graph_populator.py [hunk 2]
```diff
@@ -14,6 +14,9 @@ class GraphPopulator(object):
 
     @classmethod
     def _sanitize_text_for_query(cls, text):
+        if text is None:
+            return ''
```
Comment (jpopelka) [inline src/graph_populator.py]: Maybe `if not text:` so the same is applied also to `''` ?

File: src/graph_populator.py (hunk 1 above)
Comment (tuxdna) [reply src/graph_populator.py]: @jpopelka The check is added to `_sanitize_text_for_query()` to ensure that it works with `None` as input.

And yes you are right, we don't have to do extra check elsewhere. Will update it.

File: src/graph_populator.py (hunk 2 above)
Comment (tuxdna) [reply src/graph_populator.py]: That is concise. Will use this to update the code.

**Comment by tuxdna** [comment]
@jpopelka Update code, PTAL.

**Comment by tuxdna** [comment]
@jpopelka @sara-02 Thanks for the review :+1:

**Pull request merged**